content/concepts/consensus-network/invariant-checking.md

```diff
@@ -0,0 +1,152 @@
+# Invariant Checking
+
+This article provides a high-level overview of invariant checking, why it exist, how it works, and lists active invariants.
+
+Like many safety features, we all hope that invariant checking never actually needs to do anything. However, it can be useful to understand the XRP Ledger's invariants because they define hard limits on the XRP Ledger's transaction processing, and to recognize the problem in the unlikely event that a transaction fails because it violated an invariant check.
+
+## Introduction
+
+Invariant checking is a safety feature of the XRP Ledger. It consists of a set of checks, separate from normal transaction processing, that guarantee that certain _invariants_ hold true across all transactions.
```

Comment:
This should be the first sentence of the article. I'd move it up to replace the first sentence.